source/count.js

```diff
@@ -0,0 +1,26 @@
+import reduce from './reduce.js';
+import curry from './curry.js';
+
+
+/**
+ * Returns the number of items in a given `list` matching the predicate `f`
+ *
+ * @func
+ * @memberOf R
+ * @category List
+ * @sig (a -> Boolean) -> [a] -> Number
+ * @param {Function} predicate to match items against
+ * @return {Array} list of items to count in
+ * @example
+ *
+ *      const even = x => x % 2 == 0;
+ *
+ *      R.count(even, [1, 2, 3, 4, 5]); // => 2
+ *      R.map(R.count(even), [[1, 1, 1], [2, 3, 4, 5], [6]]); // => [0, 2, 1]
+ */
+var count = curry(function(pred, list) {
+  var r = function(a, e) { return pred(e) ? a + 1 : a; };
+  return reduce(r, 0, list);
+});
```

Comment:
I would probably choose to inline this short function inside the call to `reduce`.

```js
reduce(function(a, e) {return pred(e) ? a + 1: a}, 0, list);
```

But I don't think we've established any hard rules on that, and this version is fine.

Reply:
I don't see much difference, just a personal call. Not against your suggestion tho

Reply:
DONE